DataDog/datadog-lambda-js
# Add metric to report integration tests status

### What does this PR do?

Send a metric at the end of integration tests
- with value of 0 if success
- with value of 1 otherwise

This is done by inspecting the status of the integration tests step :

<img width="1045" alt="Screen Shot 2021-09-29 at 2 04 03 PM" src="https://user-images.githubusercontent.com/864493/135324275-abba946b-cf80-4ea1-bbc3-2f142128f637.png">

Bonus : fix a flaky test (ip for lookup is not consistent)

<img width="319" alt="Screen Shot 2021-09-29 at 1 47 37 PM" src="https://user-images.githubusercontent.com/864493/135323244-6e62bbfb-7d68-4513-82de-eee7207a6874.png">


### Motivation



Detect (via monitor + alert) when daily runs of integration tests are failing

### Testing Guidelines

Make the test fail
- you should see the alert
Fix it
- you should see the recover

### Additional Notes



### Types of Changes


### Check all that apply
